library/ssl_misc.h

```diff
@@ -254,7 +254,7 @@ uint32_t mbedtls_ssl_get_extension_mask(unsigned int extension_type);
 #if defined(MBEDTLS_SSL_PROTO_TLS1_2)
 
 /* This macro determines whether CBC is supported. */
-#if defined(MBEDTLS_SSL_HAVE_CBC)      &&                                  \
+#if defined(PSA_WANT_ALG_CBC_NO_PADDING)      &&                                  \
```

Comment:
I don't understand why we're doing this [and I don't think we should](https://github.com/Mbed-TLS/mbedtls/issues/9335#issuecomment-2218071604). It's a good thing that SSL have-algorithm guards are separate from crypto supply-algorithm guards.